src/components/map/Map.tsx

```diff
@@ -106,7 +106,7 @@ export const Map = ({
         mapPadding={{
           top: 0,
           right: 0,
-          bottom: -normalize(30),
+          bottom: normalize(device.height * 2),
```

Comment:
have you checked in report location screen and in settings for location, where we do not have a full size map?

Reply:
Yes, I have checked and confirmed that the logo is not visible. I am sharing screenshots.

|report screen|settings screen|
|--|--|
![Screenshot_20240129-144301](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/7977f8bb-0702-421e-8a57-7c31079b2fbc)|![Screenshot_20240129-144310](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/07d94a88-68a9-497a-9203-04080b1a3f12)

Reply:
but the link to some right infos on the bottom right is also gone now?

<img width="145" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/7b88a4b4-b504-424c-a1de-48103449382a">

Reply:
This still visible, but only on the iOS platform. It was not visible on Android before, it was only visible on iOS.

Reply:
i understand, all right